packages/core/src/components/dropdown/test/basic/dropdown.e2e.ts

```diff
@@ -47,4 +47,35 @@ test.describe.parallel('tds-dropdown-basic', () => {
     /* checks diff on screenshot */
     await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
   });
+  test('reset() method resets the dropdown', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    const dropdown = page.getByTestId('tds-dropdown-testid');
+
+    const dropdownButton = dropdown.getByRole('button').first();
+
+    // Get the initial placeholder text before any selection
+    const initialPlaceholderText = await dropdownButton.locator('.placeholder').innerText();
+
+    // Open the dropdown
+    await dropdownButton.click();
+
+    // Select an option from the dropdown
+    const dropdownOption = dropdown.locator('tds-dropdown-option').filter({ hasText: 'Option 1' });
+    await dropdownOption.click();
+
+    // Verify that the selected option is displayed
+    const placeholder = dropdownButton.locator('.placeholder');
+    await expect(placeholder).toHaveText('Option 1');
+
+    // Call the reset() method on the dropdown component
+    await page.evaluate(() => {
+      const dropdownnew = document.querySelector('tds-dropdown');
+      dropdownnew.reset();
+    });
+
+    // Verify that the dropdown has been reset to its initial state
+    await expect(placeholder).toHaveText(initialPlaceholderText);
+    await expect(dropdown).not.toHaveAttribute('value');
+  });
```

Comment:
I think you also need to check the value property here

Reply:
I'm encountering an issue where checking for the value breaks the test. I suspect it's because if the attribute doesn't exist, it can't retrieve any value. Do you have any suggestions on how to handle this in the code? I'm open to ideas!

Reply:
Could something like: `await expect(dropdown).toHaveJSProperty('value', null);` work? 

First check that the attribute isn't present on the element and then check that it doesn't have the jsproperty?

Reply:
Unfortunately this do not work, I do encounter a timeout with this.

Reply:
Alright, what is the stacktrace?

Reply:
Maybe verify for undefined instead?

`await expect(dropdown).toHaveJSProperty('value', undefined);`

Reply:
@mJarsater true, but as we are in:
```
private setValueAttribute = () => {
  if (!this.value || this.value?.toString() === '') {
    this.host.removeAttribute('value');
  } else {
    this.host.setAttribute('value', this.value.map((val) => val).toString());
  }
};
```
when this.value is null, it's removing the value attribute from the host element entirely, rather than setting it to "null" or undefined. Or? The value property doesn't exist on the element after reset?

Reply:
Seems like 
`await expect(dropdown).toHaveJSProperty('value', undefined);`
Works. so yes. It looks like it is set to undefined.

Reply:
@timrombergjakobsson ah yes, nice!